README.md

```diff
@@ -51,51 +51,18 @@ When you have followed the [Getting Started](https://webpack.js.org/guides/getti
 
 Otherwise `npm install --save-dev webpack-cli` or `yarn add webpack-cli --dev` will install it.
 
-# Supported arguments and commands
+## Supported arguments and commands
 
-```
-
-Available Commands
-
-  init      Initialize a new webpack configuration
-  migrate   Migrate a configuration to a new version
-  loader    Scaffold a loader repository
-  plugin    Scaffold a plugin repository
-  info      Outputs information about your system and dependencies
-  serve     Run the webpack Dev Server
-
-Options
-
-  --entry string           The entry point of your application.
-  -c, --config string       Provide path to a webpack configuration file
-  -m, --merge string       Merge a configuration file using webpack-merge
-  --progress               Print compilation progress during build
-  --silent                 Disable any output that webpack makes
-  --help                   Outputs list of supported flags
-  --defaults               Allow webpack to set defaults aggresively
-  -o, --output string      Output location of the file generated by webpack
-  --plugin string          Load a given plugin
-  -g, --global string[]    Declares and exposes a global variable
-  -t, --target string      Sets the build target
-  -w, --watch              Watch for files changes
-  -h, --hot                Enables Hot Module Replacement
-  -s, --sourcemap string   Determine source maps to use
-  --prefetch string        Prefetch this request
-  -j, --json               Prints result as JSON
-  --standard               Prints standard output
-  -d, --dev                Run development build
-  -p, --prod               Run production build
-  --version                Get current version
-  --node-args string[]     NodeJS flags
-```
+Please read [here](./packages/cli/README.md) what are the available commands and arguments.
```

Comment:
Ig this statement could be rephrased somehow.

Reply:
Please give me a suggestion. I am not good in this :)

Reply:
```suggestion
 Get to know what are the available commands and arguments [here](./packages/cli/README.md).
```

Reply:
```suggestion
Get to know what are the available commands and arguments [here](./packages/cli/README.md).
```